### PR TITLE
Add query type support

### PR DESCRIPTION
**Pull request by irmatov**
Query can have a field named "type" which defaults to "automatic".
Another possible value is "manual".

CLI result output will highlight manual query results with warning
emoji and text.

### CODE REVIEW

**Comment by irmatov** [comment]
Closes cloudquery/cloudquery-issues#85

File: pkg/config/policy.go [hunk 1]
```diff
@@ -1,9 +1,28 @@
...
+// Validate validates all contained policies.
+func (pw *PolicyWrapper) Validate() error {
+	for i, p := range pw.Policies {
+		if err := p.Validate(); err != nil {
+			return fmt.Errorf("policy %s (%d): %w", p.Name, i, err)
+		}
+	}
+	return nil
+}
+
```
Comment (roneli) [inline pkg/config/policy.go:25]: I would put this logic in the policy decode parser, see config_parser.go and config.go, when parsing and decoding we can have logic when decoding the block see provider.go

**Review by roneli — CHANGES_REQUESTED** [review]
Looks good, I would suggest one change with the normalize/validate moving it to part of the policy config parsing phase.

**Review by michelvocks — APPROVED** [review]
LGTM

File: pkg/config/policy.go (hunk 1 above)
Comment (irmatov) [reply pkg/config/policy.go]: Done

**Review by roneli — APPROVED** [review]
LGTM

**Pull request merged**